pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -257,21 +258,23 @@ t_ndarray array_slicing(t_ndarray arr, int n, ...)
     for (int32_t i = 0; i < arr.nd ; i++)
     {
         slice = va_arg(va, t_slice);
+        if (slice.end - slice.start > 1 && !offset)
+            offset = i;

```

Comment:
Are the strides properly considered if the offset is provided?

Reply:
The test seems to work fine

Reply:
https://github.com/pyccel/pyccel/pull/1299/commits/5cd16d8e7f6cae925b9446e20305fcdd89a43d29

Reply:
@EmilyBourne Can we close this conversation?